contributors/design-proposals/storage/online-grow-volume-size.md

```diff
@@ -0,0 +1,225 @@
+# Online Growing Persistent Volume Size
+
+**Authors:** [mlmhl](https://github.com/mlmhl)
+
+- [Overview](#overview)
+- [Goals](#goals)
+- [Non Goals](#non-goals)
+- [Use Cases](#use-cases)
+- [Implementation Design](#implementation-design)
+	- [Design Overview](#design-overview)
+	- [PersistentVolumeClaim Cache in Kubelet](#persistentvolumeclaim-cache-in-kubelet)
+	- [Online File System Resize in Kbuelet](#online-file-system-resize-in-kbuelet)
+		- [Volume Resize Request Detect](#volume-resize-request-detect)
+		- [File System Resizing Operation](#file-system-resizing-operation)
+
+## Overview
+
+Release 1.10 only supports offline file system resizing for PVCs, as this operation is only executed inside the `MountVolume` operation in kubelet. If a resizing request was submitted after the volume mounted, it won't be performed. This proposal intent to support online file system resizing for PVCs in kubelet.
+
+## Goals
+
+Enable users to increase size of PVC which already in use(mounted). The user will update PVC for requesting a new size.  Underneath we expect that - according kubelet will resize the file system for the PVC.
+
+## Non Goals
+
+* Offline file system resizing not included. If we found a volume need file system resizing but not mounted to the node yet, we will do nothing. This situation will be dealt with by the existing [offline file system resizing handler](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/storage/grow-volume-size.md).
+
```

Comment:
Lets reword "If we found a volume need file system resizing but not mounted to the node yet, we will do nothing." to -> "If we found a volume need file system resizing but not mounted to the node yet, we will do nothing. The file system on volume will be resized according to existing